cpu/esp32/Kconfig.esp32

```diff
@@ -24,7 +24,7 @@ config CPU_FAM_ESP32
 
     select PACKAGE_ESP32_SDK if TEST_KCONFIG
 
-    select MODULE_LIBC_GETTIMEOFDAY if TEST_KCONFIG
+    select MODULE_MALLOC_THREAD_SAFE if !MODULE_ESP_IDF_HEAP && TEST_KCONFIG
```

Comment:
This line was already added by PR #18209 commit 041fe58209c51597ec8f6fc45e11c2a6db4b9d93 but removed by mistake when Kconfig was splitted into Kconfig and Kconfig.esp32 with PR #18320 as a split-off from PR #17842

Reply:
@MrKevinWeiss Good catch, thanks for figuring it out.